atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -238,20 +232,9 @@ def parse_image_build_config(self, config_file_name):
 
         return image_name, args, {'opts': opts}
 
-    def get_filesystem_regex(self, image_name):
-        prefix = image_name
-        if self.architecture:
-            prefix = '{}.*{}'.format(prefix, self.architecture)
-
-        pattern = (r'{}.*(\.tar|\.tar\.gz|\.tar\.bz2|\.tar\.xz)$'
-                   .format(prefix))
-        filesystem_regex = re.compile(pattern, re.IGNORECASE)
-
-        return filesystem_regex
-
     def build_filesystem(self, image_build_conf):
         # Image build conf file should be in the same folder as Dockerfile
-        build_file_dir = self.workflow.source.get_build_file_path()[1]
+        build_file_dir = self.workflow.build_dir.path
```

Comment:
```suggestion
        build_file_dir = self.workflow.build_dir.any_platform.path
```
There's nothing in the root `build_dir` except the per-platform directories. The Dockerfile and image build config are in the per-platform dirs (kinda confusing, maybe the `RootBuildDir.path` attribute should be private)

Reply:
updated to use platform-specific directories.